### PR TITLE
feat(ctp): deploy NFT bridge to Kovan

### PR DESCRIPTION
**Pull request by smartcontracts**
**Description**
Adds deployment artifacts for our Kovan deployment of the NFT bridge and
all related proxy contracts. Also includes an update to
contracts-bedrock to properly export contract sources.

### CODE REVIEW

File: packages/contracts-periphery/contracts/universal/op-erc721/OptimismMintableERC721Factory.sol [hunk 1]
```diff
@@ -38,15 +38,15 @@ contract OptimismMintableERC721Factory is OwnableUpgradeable {
      * @param _bridge Address of the ERC721 bridge on this network.
      */
     constructor(address _bridge) {
-        intialize(_bridge);
+        initialize(_bridge);
```
Comment (smartcontracts) [inline packages/contracts-periphery/contracts/universal/op-erc721/OptimismMintableERC721Factory.sol:41]: Typo

**Comment by tynes** [comment]
depcheck is failing still

File: packages/contracts-periphery/contracts/universal/op-erc721/OptimismMintableERC721Factory.sol [hunk 2]
```diff
@@ -38,15 +38,15 @@ contract OptimismMintableERC721Factory is OwnableUpgradeable {
...
     }
 
     /**
      * @notice Initializes the factory.
      *
      * @param _bridge Address of the ERC721 bridge on this network.
      */
-    function intialize(address _bridge) public reinitializer(VERSION) {
```
Comment (maurelian) [inline packages/contracts-periphery/contracts/universal/op-erc721/OptimismMintableERC721Factory.sol:49]: 😬

File: packages/contracts-periphery/deploy/PeripheryProxyAdmin.ts [hunk 3]
```diff
@@ -4,15 +4,19 @@ import { DeployFunction } from 'hardhat-deploy/dist/types'
...
+      contract: 'ProxyAdmin',
+      salt: hre.ethers.utils.solidityKeccak256(
+        ['string'],
+        ['PeripheryProxyAdmin']
+      ),
+      from: deployer,
+      args: [hre.deployConfig.ddd],
+      log: true,
+    }
+  )
```
Comment (maurelian) [inline packages/contracts-periphery/deploy/PeripheryProxyAdmin.ts:19]: Where is initialize called (here and on the impl deploy script)?

**Review by maurelian — APPROVED** [review]

**Review by tynes — APPROVED** [review]

**Pull request merged**